build/templates/_converters.py.mako

```diff
@@ -7,6 +7,26 @@ from ${module_name} import visatype
 import datetime
 
 
+def convert_repeated_capabilities(repeated_capability, prefix=''):
+    # First try it as a list
+    if isinstance(repeated_capability, list):
+        rep_cap_list = [str(r) if str(r).lower().startswith(prefix.lower()) else prefix + str(r) for r in repeated_capability]
+    else:
+        # Then try it as a slice
+        try:
+            def ifnone(a, b):
+                return b if a is None else a
+            # Turn the slice into a list so we can iterate over it
```

Comment:
If you turn the slice into a list, can't you just call this function with the list recursively?

Reply:
Done